minimint/src/config.rs

```diff
@@ -96,15 +93,6 @@ impl GenerateConfig for ServerConfig {
         let (ln_server_cfg, ln_client_cfg) =
             LightningModuleConfig::trusted_dealer_gen(peers, max_evil, &(), &mut rng);
 
-        let fee_consensus = FeeConsensus {
-            fee_coin_spend_abs: minimint_api::Amount::ZERO,
-            fee_peg_in_abs: minimint_api::Amount::ZERO,
-            fee_coin_issuance_abs: minimint_api::Amount::ZERO,
```

Comment:
I vote that we keep `fee_coin_issuance_abs`. Issuing a blind signature takes computational power so I can imagine why a federation might choose to charge a fee for it.

Reply:
I've retained this field in the latest rebase. I've put the field in the mint module's config. Next to `coin_spend_abs`. Not sure if that's the right spot, please confirm.